user_docs/en/userGuide.t2t

```diff
@@ -519,6 +518,20 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Report clipboard text | NVDA+c | Reports the Text on the clipboard if there is any. |
 %kc:endInclude
 
+++ Speech modes ++[SpeechModes]
+
+There are four speech modes available.
+* Talk: NVDA will just speak normally. This is the default speech mode.
+* On-demand: NVDA will only speak when calling commands whose goal is to speak an information (e.g. report the title of the window); but it will not speak in reaction to actions such as moving the focus or the cursor.
+* Off: NVDA will not speak anything. 
+* Beeps: NVDA will replace normal speech with quiet beeps.

```

Comment:
I know "quiet" was my word, but it was a mistake.
Also, I have added some explanatory paragraphs about beeps and on-demand modes, since people are always asking what beeps mode is good for.
Lastly, I have used a hyphen for the list marker, and closed the list, since I think you will likely replace the stars used on the other items.

```suggestion
- Beeps: NVDA will replace normal speech with short beeps.
-

Beeps mode may be useful when some very verbose output is scrolling in a terminal window, but you don't care what it is, only that it is continuing to scroll; or in other circumstances when the fact that there is output is more relevant than the output itself.

The on-demand mode may be valuable when you don't need constant feedback about what is happening on screen or on the computer, but you periodically need to check particular things using review commands, etc.
Examples include while recording audio, when using screen magnification, or as an alternative to beeps mode.
```

Reply:
Thanks. I'll include it and add one more example: during a meeting or a call.